scicomp/practical-git-prs.rst

```diff
@@ -115,6 +115,28 @@ How do you make a pull request
 .. _gh-pr: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/proposing-changes-to-your-work-with-pull-requests
 .. _gl-mr: https://docs.gitlab.com/ee/README.html#merge-requests
 
+Pull request description
+~~~~~~~~~~~~~~~~~~~~~~~~
+- These days, I (rkdarst) tend to write my initial PR message into my
+  commit, then ``git-pr`` will use that when I push.  This also stores
+  it permanent.
```

Comment:
I'm not sure what's "it" that's being saved permanently here

Reply:
Fixed, it referred to the pull request description: since it's the same as the commit message, it gets recorded in the log.